docker/Dockerfile.jukebox

```diff
@@ -37,21 +37,14 @@ ENV VIRTUAL_ENV=${INSTALLATION_PATH}/.venv
 RUN python3 -m venv $VIRTUAL_ENV
 ENV PATH="$VIRTUAL_ENV/bin:$PATH"
 
-
+# Install all Python dependencies
 RUN pip install --no-cache-dir -r ${INSTALLATION_PATH}/requirements.txt
 
-ENV ZMQ_TMP_DIR libzmq
-ENV ZMQ_VERSION 4.3.5
-ENV ZMQ_PREFIX /usr/local
-
-RUN [ "$(uname -m)" = "aarch64" ] && ARCH="arm64" || ARCH="$(uname -m)"; \
-    wget https://github.com/pabera/libzmq/releases/download/v${ZMQ_VERSION}/libzmq5-${ARCH}-${ZMQ_VERSION}.tar.gz -O libzmq.tar.gz; \
-    tar -xzf libzmq.tar.gz -C ${ZMQ_PREFIX}; \
-    rm -f libzmq.tar.gz;
-
-RUN export ZMQ_PREFIX=${PREFIX} && export ZMQ_DRAFT_API=1
-RUN pip install -v --no-binary pyzmq pyzmq
+# Install pyzmq Python dependency separately
+ENV ZMQ_PREFIX ${VIRTUAL_ENV}
+ENV ZMQ_DRAFT_API 1
+COPY --from=libzmq /usr/local ${ZMQ_PREFIX}
```

Comment:
Copying must be done before the user is switched for sufficient access rights.
See #2305

Reply:
Strange, it works perfectly for me ?!

Reply:
But I haven't tested other environments yet! :-)

Reply:
ah ok, it seems like the COPY command is always related to `root` unless `--chown` is used.
So this differes from the report of @notapirate in #2305.

For readability it might still be better to change order :)

Reply:
I'd like to keep it there as it belongs together as an entity

Reply:
i rather meant moving the whole pyzmq setup

Reply:
But it can't go above the venv definition and then there is only the python requirements part left above it. I'd keep it that way, it works great.

Else I don't really get your proposal ;-)

Reply:
Now that the path changed it could  ^^

Reply:
The pip command needs to stay where it is, in order to be installed into the venv. If I move the copy command and the ENV definition higher up, I separate concerns (pyzmq) for the reason of merging another concern (copy)

Reply:
my mistake, misunderstood the last comment about the venv. 
You are right, it's better kept this way 👍